src/SignalRServiceExtension/Client/AzureSignalRClient.cs

```diff
@@ -29,6 +29,7 @@ internal class AzureSignalRClient : IAzureSignalRSender
 
         private readonly IServiceManagerStore serviceManagerStore;
         private readonly string connectionStringKey;
+        private ValueTask<IServiceHubContext> HubContext => serviceManagerStore.GetOrAddByConnectionStringKey(connectionStringKey).GetAsync(HubName);
```

Comment:
how about change it to `ValueTask<IInternalServiceHubContext> GetInternalServiceHubContext()` lok slae you only need `IInternalServiceHubContext`